WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormListViewModel.swift

```diff
@@ -5,14 +5,27 @@ import protocol Storage.StorageManagerType
 /// View model for ShippingLabelsCustomsFormList
 ///
 final class ShippingLabelCustomsFormListViewModel: ObservableObject {
+    /// Whether packages multiple packages are found.
```

Comment:
nit: Small typo here, suggested fix:

```suggestion
    /// Whether multiple packages are found.
```